cookbook/system.md

```diff
@@ -121,34 +121,5 @@ Output
 
 Notes:
 
-- `kill` is Linux/Unix specific command, it is not built-in to nu.
-- filtering with the `where` command, as shown above, is case-sensitive.
-
----
-
-### Pipeline content to clipboard
```

Comment:
what about mentioning `std clip` instead?

Reply:
might better be in the [*Standard library* section](https://www.nushell.sh/book/standard_library.html) :thinking:

Reply:
I don't know how to document stdlib functionality properly, because I don’t really understand the stdlib. I’ll leave that to someone else for now